### ISSUE TITLE
Class type of a resolver returned value is ignored

### ISSUE THREAD

**Issue by maksimkurb** [nestjs/graphql#241]
## Current behavior
`@nestjs/graphql` doesn't check the class type of returned value in resolver and throws `Abstract type ... must resolve to an Object type at runtime for field ...`

## Expected behavior
When I return an instance of a class in resolver, it should automatically resolve type of the instance by comparing it's class type to the all imlementations of returning interface type as `type-graphql` does.

## Minimal reproduction of the problem with instructions
```ts
  @InterfaceType()
  interface IPerson { ... } // our interface

  @ObjectType({ implements: [IPerson] })
  class StudentPerson implements IPerson { ... } // our impl

  @Resolver(() => IPerson)
  export class PersonResolver {
    @Query(() => IPerson)
    public async request(): Promise<IPerson> {
      return new StudentPerson(); // <-- @nestjs/graphql does not check the class type of an instance
    }
  }
```

## What is the motivation / use case for changing the behavior?
Not writing `resolveType` functions when we can check type by checking instance class type

## Temp fix:
Write resolveType function in resolver which returns class name (will not work if you use custom ObjectType names):
```ts
class SomeResolver {
  @ResolveProperty('__resolveType')
  public resolveType(value: Request) {
    return Object.getPrototypeOf(value).constructor.name
  }
}
```

## Environment

<pre><code>
Nest version: ^6.1.1
 
For Tooling issues:
- Node version: 11.14.0
- Platform:  Windows

Others:
- @nestjs/graphql version: 6.0.5
- type-graphql version: 0.17.3
I'm using code-first approach
</code></pre>

**Comment by kamilmysliwiec** [comment]
It should be fixed in the @next version (`$ npm i @nestjs/graphql@next`). You will also have to update `@nestjs/core` to 6.2.0 (`$ npm i @nestjs/core@6.2.0`)

**Comment by kamilmysliwiec** [comment]
Published as `6.2.0` (`$ npm i @nestjs/graphql@latest`). You will also have to update `@nestjs/core` to 6.2.0 (`$ npm i @nestjs/core@latest`)
